src/queueList/Main.java

```diff
@@ -0,0 +1,45 @@
+package queueList;
+
+import javax.swing.*;
+
+public class Main {
+
+    public static void main(String[] args) {
+        int option = 0, inputData = 0;
+        Queue queue = new Queue();
+        do {
+
+            try {
+                option = Integer.parseInt(JOptionPane.showInputDialog(null, "Options: \n\n"
+                        + "1. Insert \n\n"
+                        + "2. Extract \n\n"
+                        + "3. Display Content \n\n"
+                        + "4. Exit \n\n"));
+
+                switch (option) {
+                    case 1:
+                        inputData = Integer.parseInt(JOptionPane.showInputDialog(null, "Please Enter a Value"));
+                        queue.Insert(inputData);
+                        break;
+                    case 2:
+                        if (!queue.EmptyQueue()) {
+                            JOptionPane.showMessageDialog(null, "A node was extracted with the value: " + queue.Extract());
+                        } else {
+                            JOptionPane.showMessageDialog(null, "The queue is Empty");
+                        }
+                        break;
+                    case 3:
+                        queue.DisplayContent();
+                        break;
+                    case 4:
+                        option = 4;
+                        break;
+                    default:
+                        JOptionPane.showMessageDialog(null, "Invalid Option");
+                        break;
+                }
+            } catch (NumberFormatException e) {
+            }
+        } while (option != 4);
+    }
+}
```

Comment:
Please avoid using UI components in these exercises. Use the terminal instead.

In this queue exercise, the idea is to use the methods `add` and `remove` to see how a queue is a FIFO (First In First Out) data structure.

See:
* https://en.wikipedia.org/wiki/Queue_(abstract_data_type)
* https://docs.oracle.com/javase/8/docs/api/java/util/Queue.html